loop-client/deployment_tests/test_deployment.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import demjson
+
+MAIN_URL = 'https://call.stage.mozaws.net'
```

Comment:
Version info should be queried directly from the URL endpoints (rather than hard-coded in the tests), though I don't believe this is currently enabled for loop-client.  I still wouldn't hard-code it here, as you'd need to update your test repo to change version with every deploy.  In any event, the version number should be reflected via the stack-check (once the version has been exposed).

Also, all endpoints are going to eventually be passed to us by Ops, but for now our convention is to put them in a manifest file like so:
services-test/loop-client/manifest.ini
Create a file w/ this format (test envs as sections), then add it to the loop-client sub dir and parse those values for your args.

[dev]
root = https://call-dev.stage.mozaws.net
[stage]
root = https://call.stage.mozaws.net
[prod]
root = https://hello.firefox.com
etc.